docs/installation-configuration.md

```diff
@@ -41,7 +14,9 @@ To provision resources with the Pulumi Nuage provider,  you can set up the sdk o
 * Go: [`nuage`](https://github.com/nuage-studio/pulumi-nuage/tree/master/sdk/go/nuage)
 * .NET: [`Pulumi.Nuage`](https://github.com/nuage-studio/pulumi-nuage/tree/master/sdk/dotnet)
 
+## Credentials
 
+Pulumi Nuage components contains AWS Components from `pulumi-aws`and you can refer to corresponding docs for settting AWS credentials.  [correspnding docs](https://www.pulumi.com/registry/packages/aws/installation-configuration/)
```

Comment:
One extra t in settings

Reply:
@burakgzt if you do a new release for the .NET issue maybe you could fix this small typo as well ? 🙏